admin_solutions/master_node_config.adoc

```diff
@@ -0,0 +1,396 @@
+
+. Open the configuration file you want to modify, which in this case is the master configuration file:
++
+----
+$ vim /etc/origin/master/master-config.yaml
+----
++
+. Add the following new variables to the `*identityProviders*` stanza of the file:
++
+----
+oauthConfig:
+  ...
+  identityProviders:
+  - name: my_htpasswd_provider
+    challenge: true
+    login: true
+    mappingMethod: claim
+    provider:
+      apiVersion: v1
+      kind: HTPasswdPasswordIdentityProvider
+      file: /path/to/users.htpasswd
+----
+. Save your changes and close the file.
+. Restart the master for the changes to take effect:
++
+----
+$ systemctl restart atomic-openshift-master
+----
+
+These simple steps are all that is required to manually modify the master and node configuration files.
+For information on all available options, see the
+link:../admin_solutions/master_node_config.html#master-config-options[master configuration] and
+link:../admin_solutions/master_node_config.html#node-config-options[node configuration] options reference tables.
+
+=== Master Configuration File Options
+
+[[master-config-options]]
+[cols="1,4"]
+.Master Configuration File Options
+|===
+|Option |Description
+
+|`*servingInfo*`
+|Describes how to start serving.
+
+|`*corsAllowedOrigins*`
+|Specifies the host name to use to access the API server from a web application.
+
+|`*apiLevels*`
+|A list of API levels that should be enabled on startup; for example, `v1beta3` and `v1`.
+
+|`*masterPublicURL*`
+|The URL that clients use to access the OpenShift API server.
+
+|`*controllers*`
+|A list of the controllers that should be started. If set to `none`, then no controllers will start automatically. The default value is `\*` which will start all controllers. When using `*`, you may exclude controllers by prepending a `-` in front of the controller name. No other values are recognized at this time.
+
+|`*pauseControllers*`
+|When set to `true`, this instructs the master to not automatically start controllers, but instead to wait until a notification to the server is received before launching them.
+
+|`*controllerLeaseTTL*`
+|Enables controller election, instructing the master to attempt to acquire a lease before controllers start, and renewing it within a number of seconds defined by this value. Setting this value as a non-negative forces `*pauseControllers=true*`. The value default is off (`0`, or omitted) and controller election can be disabled with `-1`.
+
+|`*admissionConfig*`
+|Contains link:../install_config/configuring_admission_control.html[admission control plug-in] configuration. OpenShift has a configurable list of admission controller plug-ins that are triggered whenever API objects are created or modified. This option allows you to override the default list of plug-ins; for example, disabling some plug-ins, adding others, changing the ordering, and specifying configuration. Both the list of plug-ins and their configuration can be controlled from Ansible.
```

Comment:
Same comment here about the admission control plug-in. Could link to:

../architecture/additional_concepts/admission_controllers.html

The gist of the why things changed up is here: https://github.com/openshift/openshift-docs/pull/2023